frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/confirmation.tsx

```diff
@@ -189,7 +268,7 @@ export default function ApplyFlowConfirm() {
           <p>
             {t('confirm.app-code-is')}
             <br />
-            <strong>{submissionInfo.confirmationCode}</strong>
+            <strong>{submissionInfo?.confirmationCode ?? 'confirmationCode'}</strong>
```

Comment:
Does it make sense to render `confirmationCode` if there is no confirmation code? I feel like if the confirmation code is required to render, and it isn't defined, you should throw.

@sebastien-comeau agree/disagree?